lib/utils.js

```diff
@@ -64,15 +66,20 @@ const utils = {
     utils.log(`${c.redBright('Error')} ${err && err.message ? err.message : err}`, newline);
   },
 
-  loadUserConfig: async () => {
+  loadUserConfig: async (key) => {
     const f = resolveUserConfig();
     await ensureFile(f);
     const config = yaml.safeLoad(await readFile(f)) || {};
     if (isEmpty(config)) {
       utils.logError('User Configuration file missing, run losant login to generate this file.');
       process.exit(1);
     }
-    return config;
+    if (has('apiToken', config)) {
+      config[DEFAULT_URL] = { apiToken: config.apiToken };
```

Comment:
you are going to get a weird thing when the old file migrates if the old file had a token for the non default url.....